safedelete/models.py

```diff
@@ -1,7 +1,6 @@
-from collections import Counter
-from collections import defaultdict
-from itertools import chain
 import warnings
+from collections import Counter, defaultdict
+from itertools import chain
```

Comment:
can't explain why this change happened. Any ideas?

Reply:
Do you have isort automatically running on save?

Reply:
isort is actually always called with tox as it is configured. That's probably why.